debian-pkg/debian/patches/0001-CMakeLists.txt_tests-CMakeLists.txt_find-check-via-p.patch

```diff
@@ -1,13 +1,21 @@
-Description: Search for libcheck via pkg-config instead of cmake; use static version.
-Author: Mihai Moldovan <ionic@ionic.de>
+From 789f4493ce90757d75888bb716ea7bd15f3846ed Mon Sep 17 00:00:00 2001
+From: Mihai Moldovan <ionic@ionic.de>
+Date: Tue, 24 Nov 2020 11:26:00 +0100
+Subject: [PATCH] CMakeLists.txt_tests-CMakeLists.txt_find-check-via-pkg-config
```

Comment:
Use previous `Description` field for `Subject`. This applies to other patches too.

Reply:
Yes. I should check more options on `git quiltimport` to see if there is better option to preserver quilt patch original format.